sci-astronomy/stellarium/stellarium-1.2.ebuild

```diff
@@ -0,0 +1,190 @@
+		verify-sig? ( https://github.com/Stellarium/stellarium/releases/download/v${PV}/stellarium_user_guide-${PV}-1.pdf.asc )
+	)
+	stars? (
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_4_1v0_2.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_5_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_6_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_7_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_8_2v0_1.cat
+	)"
+
+LICENSE="GPL-2+ SGI-B-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~riscv"
+IUSE="debug deep-sky doc gps media nls qt6 +scripting +show-my-sky stars telescope test webengine +xlsx"
+# Qt6 QAudioOutput crashes on startup; qt 6.4.0
+# https://bugreports.qt.io/browse/QTBUG-108221
+REQUIRED_USE="|| ( !media !qt6 )"
+
+# Python interpreter is used while building RemoteControl plugin
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-lang/perl
+	doc? ( app-doc/doxygen[dot] )
+	nls? (
+		!qt6? ( dev-qt/linguist-tools:5 )
+		qt6? ( dev-qt/qttools:6[linguist] )
+	)
+	verify-sig? ( sec-keys/openpgp-keys-stellarium )
+"
+RDEPEND="
+	media-fonts/dejavu
+	sys-libs/zlib
+	gps? ( sci-geosciences/gpsd:=[cxx] )
+	media? ( virtual/opengl )
+	!qt6? (
+		dev-qt/qtcharts:5
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5=
+		dev-qt/qtnetwork:5
+		dev-qt/qtwidgets:5
+		sci-astronomy/calcmysky:=[qt5]
+		gps? (
+			dev-qt/qtpositioning:5
+			dev-qt/qtserialport:5
+		)
+		media? (
+			dev-qt/qtmultimedia:5[widgets]
+			dev-qt/qtopengl:5
+		)
+		scripting? ( dev-qt/qtscript:5 )
+		telescope? ( dev-qt/qtserialport:5 )
+		webengine? ( dev-qt/qtwebengine:5[widgets] )
+		xlsx? ( dev-libs/qxlsx:=[qt5] )
+	)
+	qt6? (
+		dev-qt/qtbase:6=[gui,network,widgets]
+		dev-qt/qtcharts:6
+		sci-astronomy/calcmysky:=[qt6]
+		gps? (
+			dev-qt/qtpositioning:6
+			dev-qt/qtserialport:6
+		)
+		media? (
+			dev-qt/qtmultimedia:6[gstreamer]
+		)
+		scripting? ( dev-qt/qtdeclarative:6 )
+		telescope? ( dev-qt/qtserialport:6 )
+		webengine? ( dev-qt/qtwebengine:6[widgets] )
+		xlsx? ( dev-libs/qxlsx:=[qt6] )
+	)
+	telescope? ( sci-libs/indilib:= )
+"
+DEPEND="${RDEPEND}
+	!qt6? (
+		dev-qt/qtconcurrent:5
+		test? ( dev-qt/qttest:5 )
+	)
+	qt6? ( dev-qt/qtbase:6=[concurrent] )
+"
+
+RESTRICT="!test? ( test )"
+
+PATCHES=(
+	"${FILESDIR}/stellarium-0.20.3-unbundle-zlib.patch"
+	"${FILESDIR}/stellarium-0.22.2-ccache.patch"
+)
+
+VERIFY_SIG_OPENPGP_KEY_PATH=${BROOT}/usr/share/openpgp-keys/stellarium.asc
+src_unpack() {
+	# stars-2.0 are not signed
+	pushd "${DISTDIR}" > /dev/null || die
+	verify-sig_verify_detached \
+		${P}.tar.xz{,.asc}
+	use deep-sky && verify-sig_verify_detached \
+		${PN}-dso-catalog-${MY_DSO_VERSION}.dat{,.asc}
+	use doc && verify-sig_verify_detached \
+		stellarium_user_guide-${PV}-1.pdf{,.asc}
+	popd > /dev/null || die
+	default_src_unpack
```

Comment:
You probably need some sort of `if use verify-sig; then` condition in here. Otherwise the main keys aren't downloaded.

Reply:
you're right, fixed